src/views/FarmAuction/hooks/useWhitelistedAddresses.ts

```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from 'react'
+import { useFarmAuctionContract } from 'hooks/useContract'
+import { getBidderInfo } from 'config/constants/farmAuctions'
+import { FarmAuctionBidderConfig } from 'config/constants/types'
+import { AUCTION_WHITELISTED_BIDDERS_TO_FETCH } from 'config'
+
+const useWhitelistedAddresses = () => {
```

Comment:
Used to show WhitelistedBiddersModal
![Screenshot 2021-08-02 at 2 50 22](https://user-images.githubusercontent.com/81824236/127788963-6cd46e99-468a-40e4-874e-8ca9f3654a81.png)